contracts/voting_escrow_delegation/src/helpers.rs

```diff
@@ -2,19 +2,20 @@ use crate::state::{Config, Token, DELEGATED};
 use crate::ContractError;
 use astroport_governance::utils::calc_voting_power;
 use astroport_governance::voting_escrow::get_lock_info;
-use astroport_governance::voting_escrow_delegation::{
-    DELEGATION_MAX_PERCENT, DELEGATION_MIN_PERCENT,
-};
-use cosmwasm_std::{Addr, Deps, Order, QuerierWrapper, StdError, StdResult, Uint128};
+use cosmwasm_std::{Addr, Decimal, Deps, Order, QuerierWrapper, StdError, StdResult, Uint128};
+
+pub(crate) const MAX_BPS_AMOUNT: u16 = 10000u16;
+pub(crate) const MIN_BPS_AMOUNT: u16 = 1u16;
```

Comment:
```suggestion
const MAX_BPS_AMOUNT: u16 = 10000u16;
const MIN_BPS_AMOUNT: u16 = 1u16;
```
Consts can be private